### ISSUE TITLE
New diagram on preliminary.istio.io is of questionable accuracy

### ISSUE THREAD

**Issue by linsun** [istio/istio.io#9444]
hey @craigbox opening this for discussion:

https://preliminary.istio.io home page

This diagram shows metrics going through istio cp... is that accurate with telemetry v2?

**Comment by craigbox** [comment]
This is meant to be a conceptual diagram, not an actual "data flows through these boxes".

The metrics backend is a third party to Istio, but one might think of it as part of the backend of one's service mesh. It's definitely not part of the data plane.

Very open to ideas on how to get across how simple the architecture is without having to add extra boxes.  Considering having the arrow pointing out to, ahem, "the cloud".

**Comment by craigbox** [comment]
Having discussed with the team for now, we're going to leave it; the goal is to give a 10,000 ft overview.

**Comment by linsun** [comment]
I don't agree closing this.  The diagram shows the metrics flow to Istio control plane, which is wrong and made me to think of our old mixer model.

**Comment by linsun** [comment]
can we just add a box for metrics service/service?  (I apologize for not replying sooner!)

**Comment by craigbox** [comment]
The point is to have as few things in this picture as possible.   More correct diagrams exist in places like https://preliminary.istio.io/latest/docs/ops/deployment/architecture/.

Someone who has no conceptual familiarity with Istio should be able to get a rough understanding that (a) the control plane instructs the data plane (b) Istio does telemetry.

We are thinking of changing "monitoring" to "telemetry" but no other changes are planned.

**Comment by linsun** [comment]
We only has metrics on the diagram atm, I don't see `monitoring`.  Could we just have metrics with an arrow point out (but not actually point to Istiod) so users know it can integrate with a metrics backend service?
